client/src/slices/user.jsx

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const initialState = { email: '', password: '' };
```

Comment:
Do you really need to store the email and the passport in the store? what data you need in other components?